docs/apis/internal/chainer.md

```diff
@@ -0,0 +1,85 @@
+# Chainer API
+
+This API is for communication between the Seldon Scheduler and the Seldon dataflow component which manages pipelines.
+
+## Proto Definition
+
+```proto
```

Comment:
similar to comment above, this cannot be text in the docs which duplicates ground-truth protobuf files.

Reply:
resolved. see live changes here https://docs.seldon.ai/seldon-core-v2/apis/internal/chainer